litestar/middleware/logging.py

```diff
@@ -305,7 +305,7 @@ class LoggingMiddlewareConfig:
             Thus, re-arranging the log-message is as simple as changing the iterable.
         -  To turn off logging of responses, use and empty iterable.
     """
-    middleware_class: type[LoggingMiddleware] = field(default_factory=lambda: LoggingMiddleware)
+    middleware_class: type[LoggingMiddleware] = LoggingMiddleware
```

Comment:
How about use `field` function for consistency.
```suggestion
    middleware_class: type[LoggingMiddleware] = field(default=LoggingMiddleware)
```

Reply:
Makes sense. Thanks for the suggestion!